src/base/Macros.hh

```diff
@@ -94,8 +94,8 @@
  * or https://msdn.microsoft.com/en-us/library/1b3fsfxw.aspx=
  *
  * \note This macro is usually unused; instead, the macro \c
- * CHECK_UNREACHABLE defined in DBC.hh should be used instead (to provide
- * a more detailed error message in case the point *is* reached).
+ * CELER_ASSERT_UNREACHABLE() defined in DBC.hh should be used instead (to
```

Comment:
Did you mean:
```suggestion
 * CELER_ASSERT_UNREACHABLE() defined in base/Assert.hh should be used instead (to
```
?

Reply:
Hah, nice catch -- a holdover from the file in Shift in which I originally wrote the macro 😅